### ISSUE TITLE
Unable to boot up RPi 3b+

### ISSUE THREAD

**Issue by mharishankar** [kkamagui/shadow-box-for-arm#4]
Hi,
Is your build intended to work with the Pi model 3b+? I followed your instructions exactly as is multiple times from scratch, but the Pi doesn't boot up and gives me red flashing light (4 flashes in a row). I also get no HDMI output though monitor is plugged in.

Thanks!

**Comment by kkamagui** [comment]
@mharishankar , I guess you used the wrong firmware or OS image. This repository is for RPI 3, not RPI 3B or 3B+. So, that's the reason I think. :)

**Comment by mharishankar** [comment]
Got it. I'm planning on replaying your changes to the

**Comment by mharishankar** [comment]
Got it, thanks for the clarification! I'll change the firmware + kernel image. 

Can you also clarify why rpi3.mk in build/ seems to repeat the same targets multiple times? Eg make all executes linux as well as update_rootfs, but update_rootfs executes filelist-tee which executes linux again. Is there any good reason for doing this?

**Comment by kkamagui** [comment]
@mharishankar , hmm... I'm not sure, but is update_rootfs just copies the Linux kernel executable? As I know, update_rootfs makes a root filesystem. So it just copies the kernel into the filesystem. :)

**Comment by mharishankar** [comment]
Yup I removed the redundant remakes of linux from update_rootfs and some other places, and make all works just fine. Also, I was able to get everything working on a Pi 3B without major issues! (Pi 3B+ didnt work, and I didnt try with Pi 3; looks like you're referencing 3-b.dtb in build/ so I think your repo is currently configured for 3B).

**Comment by mharishankar** [comment]
I'm confused about executing the batch_ima_sign script. The script simply signs the files in the directories you've specified without storing the signature or any output anywhere. Whats the point of running this script? Can you please clarify?

**Comment by kkamagui** [comment]
@mharishankar , it's fine. 👍 I hope everything goes well.
